src/extension.ts

```diff
@@ -430,11 +430,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         return new LanguageClient('Phan Language Server', serverOptions, clientOptions);
     };
 
+    const languageClients = analyzedProjectDirectories.map(createClient);
+
+    async function restartLanguageClients(): Promise<void> {
+        for (const client of languageClients) {
+            await client.stop();
+            context.subscriptions.push(client.start());
```

Comment:
This seems unnecessary given how client.restart is implemented, it looks like only one disposable is needed per client.